connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/PluginUtils.java

```diff
@@ -128,6 +128,7 @@
             + "|file\\..*"
             + "|converters\\..*"
             + "|storage\\.StringConverter"
+            + "|rest\\.basic\\.auth\\.extenstion\\.BasicAuthSecurityRestExtension"
```

Comment:
typo in: `extenstion` 

Unfortunately the typo is in the package and needs to be corrected. 

Mind adding a unit test case in: `PluginUtilsTest` as well?